### DISCUSSION TITLE
Creating a q-field of q-chips and q-input

### DISCUSSION THREAD

**Discussion by jmsuresh** [quasarframework/quasar#11530] [General - Components / Directives / etc] [answered]
I am trying to create an input field similar to how tags are entered in the Stackoverflow question editor.

```
    <q-field label="Author(s)" stack-label>
      <div>
        <q-chip v-for="(book_author, i) in book_authors" :key="i">{{ book_author.name }}</q-chip>
        <q-input
          v-model="new_author_name"
          lazy-rules
          @keydown.enter="add_author"
        />
      </div>
    </q-field>
```

I could not get the styling work. I want the input field to appear after the last chip as if the chips are inside the q-input. Any suggestions?

**Comment by pdanpdan** [comment]
codepen please

**Reply by jmsuresh** [reply]
Please find here: https://codepen.io/jm-suresh/pen/gOGbXYx

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/xxXbpxr?editors=1100
